syne_tune/optimizer/schedulers/searchers/bayesopt/contributed/estimator.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you think of a different name than `contributed`?

This sounds like we discourage contributions on the BO side.